ansible_builder/utils.py

```diff
@@ -3,13 +3,27 @@
 import os
 import filecmp
 import shutil
+import logging
 
 from .colors import MessageColors
 
 
+logger = logging.getLogger(__name__)
+logging_levels = {
+    '0': logging.ERROR,
+    '1': logging.WARNING,
+    '2': logging.INFO,
+    '3': logging.DEBUG,
+}
+
+
+def configure_logger(verbosity):
+    logging.basicConfig(stream=sys.stdout, level=logging_levels[str(verbosity)], format='%(message)s')
+
+
 def run_command(command, capture_output=False, allow_error=False):
-    print(MessageColors.HEADER + 'Running command:' + MessageColors.ENDC)
-    print(MessageColors.HEADER + '  {0}'.format(' '.join(command)) + MessageColors.ENDC)
+    logger.debug(MessageColors.HEADER + 'Running command:' + MessageColors.ENDC)
+    logger.debug(MessageColors.HEADER + '  {0}'.format(' '.join(command)) + MessageColors.ENDC)
```

Comment:
Big picture, wouldn't it be better to have the logging method apply the color, as opposed to having to put it inside the string passed into the log call?

There are plenty of ways you could do this, and the safest I think would be to, instead of calling `logger.debug`, call utility methods that you introduce like `debug.v("msg")`, where it then maps the method called to both the color and the logging method.

But if you wanted to use the python logging methods debug/info/warn/etc. then you could still put the color in the format statement, because you're _already_ constructing `format='%(message)s'`. Because we're not supporting python 2, you actually have the ability to do the dictionary lookup _inside_ the f-string, like:

https://github.com/borntyping/python-colorlog/blob/ed2d85cb0e9e175add18230a4720f1e0739c9e59/colorlog/colorlog.py#L25

That would also allow you to get rid of the re-typing of `MessageColors.HEADER` for every message